app/models/agent.rb

```diff
@@ -129,12 +129,18 @@ def reload
   end
 
   def new_event_expiration_date
-    keep_events_for > 0 ? keep_events_for.days.from_now : nil
+    if keep_events_for == 60
```

Comment:
After doing the migration, should I commit the changes to `db/schema.rb` as well?